src/utils/V3Utils.ts

```diff
@@ -83,6 +85,11 @@ export function isSlowFill(fill: Fill): boolean {
   return isV2Fill(fill) ? fill.updatableRelayData.isSlowRelay : fill.relayExecutionInfo.fillType === FillType.SlowFill;
 }
 
+// If needed, this can be made generic in the future.
+export function isSlowFillRequest(fill: Fill | SlowFillRequest): fill is SlowFillRequest {
+  // Relayer field exists in V2Fill, V3Fill, but not SlowFillRequest.
+  return (fill as Fill).relayer === undefined;
```

Comment:
This might be a cleaner way of handling this:

```suggestion
export function isSlowFillRequest(fill: Fill | SlowFillRequest): fill is SlowFillRequest {
  return !isV2Fill(fill) && !isV3Fill(fill);
```